templates/keyfile

```diff
@@ -0,0 +1,9 @@
+{% if mariadb_encryption_keygen_auto %}
+{% for data in range(1,mariadb_encryption_keygen_count+1) %}
+{{ data }};{{ lookup("ansible.builtin.password", "/dev/null", length=64, chars=["abcdef", "digits"], seed=ansible_facts['default_ipv4']['macaddress'] + [data] | join('')) }}
+{% endfor %}
+{% else %}
+{% for key,value in mariadb_encryption_keys.items() %}
+{{ key }};{{ value }}
+{% endfor %}
+{% endif %}
```

Comment:
This file only being used when mariadb_encryption_keygen_auto is set to false, the else part of this file is never used, therefore I think it the never executed parts could be removed.